src/libraries/System.Formats.Tar/src/System.Formats.Tar.csproj

```diff
@@ -59,7 +59,6 @@
     <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.FChMod.cs" Link="Common\Interop\Unix\System.Native\Interop.FChMod.cs" />
     <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.Link.cs" Link="Common\Interop\Unix\System.Native\Interop.Link.cs" />
     <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.MkFifo.cs" Link="Common\Interop\Unix\System.Native\Interop.MkFifo.cs" />
-    <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.Permissions.cs" Link="Common\Interop\Unix\Interop.Permissions.cs" />
```

Comment:
@bartonjs @eerhardt after this PR is merged, I can get rid of `TarFileMode` and use `UnixFileMode` instead. Agreed?

Reply:
Yes, the plan is to replace it. https://github.com/dotnet/runtime/pull/69980#discussion_r888117453